src/plugins/workspace/public/components/workspace_form/add_collaborator_button.tsx

```diff
@@ -85,7 +95,18 @@ export const AddCollaboratorButton = ({
       ...permissionSettings,
       ...addedSettings.map(({ collaboratorId, ...rest }) => rest),
     ];
-    await handleSubmitPermissionSettings(newPermissionSettings as WorkspacePermissionSetting[]);
+    const result = await handleSubmitPermissionSettings(
+      newPermissionSettings as WorkspacePermissionSetting[]
+    );
+    if (result?.success) {
+      if (notifications) {
+        notifications.toasts.addSuccess({
+          title: i18n.translate('workspace.workspaceDetail.collaborator.add.success', {
```

Comment:
nit: should we remove workspaceDetail from the key `workspace.collaborator.add.success`